DNN Platform/Library/Entities/Urls/RewriteController.cs

```diff
@@ -1392,7 +1392,11 @@ internal static void GetUrlWithQuerystring(HttpRequest request, Uri requestUri,
                 absoluteUri = absoluteUri.Replace(queryString, "");
             }
             absoluteUri = HttpUtility.UrlDecode(absoluteUri); //decode the incoming request
-            string rewritePath = GetTabFromDictionary(absoluteUri, queryStringCol, settings, result, parentTraceId);
+
+            // below 810 check is taking care of lanugage code
+            var absoluteUriWithLanguageCode = Regex.Replace(absoluteUri, $"\\/{result.CultureCode}", string.Empty, RegexOptions.IgnoreCase);
```

Comment:
I think this variable should be called `*WithoutLanguageCode`?

Reply:
Are we certain that the regular expression will always match successfully? What I mean is a possibility that the URI will contain a language code different from the one in `result.CultureCode`?